virttest/utils_test/__init__.py

```diff
@@ -251,6 +251,72 @@ def update_boot_option(
             session.close()
 
 
+def update_vm_default_kernel(
+    vm,
+    kernel_version,
+    reboot=True,
+    timeout="",
+    guest_arch_name="x86_64",
```

Comment:
This one should also not have a default value, s. comment above. This will avoid failures due to 'copy-paste' which we often use to discover new functions.